controllers/chat/group/changeChannelDetail.js

```diff
@@ -25,6 +26,8 @@ const changeChannelDetail = async (req, res) => {
 
   if (queriedChannel?.length < 1) return ErrorResponse.e404(res, 'Group not found');
 
+  let all_members = queriedChannel[0].data.better_channel_member.map((member) => member.user.id);
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The extraction of all members from the queried channel is a new addition. Make sure that `queriedChannel[0].data.better_channel_member` always exists and is an array before calling `map`. If not, it could lead to a runtime error.
```diff
+ let all_members = queriedChannel[0].data.better_channel_member.map((member) => member.user.id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->